examples/feature_demo/gltf_viewer.py

```diff
@@ -84,6 +82,8 @@
 background.visible = False
 scene.add(background)
 
+scene.add(gfx.Background.from_color((0.1, 0.1, 0.1, 1)))
```

Comment:
Can a scene have two backgrounds?

Reply:
One is the environment background (skybox), and the other is a solid color background. My expectation is that when the background is checked, the skybox background will be displayed, and when unchecked, a default solid color background will appear. Adding both backgrounds seems to work correctly (when displayed together, the skybox will override the solid color background).

For reference: The reason for this approach is that I encountered an issue while implementing Physically-Based Transparency. When generating the transmission light sampling texture, the background in the framebuffer should not be transparent. I will continue to explore solutions in #974.

Reply:
Locally this works for me as described :+1: cool!